src/ui/server/content-strings/fields/insurance/your-buyer/index.ts

```diff
@@ -1,5 +1,9 @@
 export const FIELDS = {
   YOUR_BUYER: {
+    BUYER_ORGANISATION: {
+      ID: 'organisation',
+      LABEL: "Buyer's company or organisation name",
+    },
```

Comment:
We shouldn't need ID's in field content strings files, just the content of the field - example: https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/content-strings/fields/index.ts

Reply:
its really confusing few places has id's [https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/content-strings/fields/insurance/policy-and-exports/index.ts].

I believe we need ids With out ids how can you 

how can you pass it          'data-cy': YOUR_BUYER.BUYER_ORGANISATION.ID + '-label'?